forest/src/cli/config.rs

```diff
@@ -51,13 +89,13 @@ impl Default for DaemonConfig {
 #[derive(Serialize, Deserialize, PartialEq, Default)]
 #[serde(default)]
 pub struct Config {
+    pub log: LogConfig,
     pub client: Client,
     pub rocks_db: forest_db::rocks_config::RocksDbConfig,
     pub network: Libp2pConfig,
     pub sync: SyncConfig,
     pub chain: Arc<ChainConfig>,
     pub daemon: DaemonConfig,
-    pub log: LogConfig,
```

Comment:
Is the configuration prettier with the log config at the top?

Reply:
that's to evade `ValueAfterTable` error

Reply:
What error is that?

Reply:
ValueAfterTable
[−]
All values in a TOML table must be emitted before further tables are emitted. If a value is emitted after a table then this error is generated.

All structs and maps are considered as tables

Reply:
I don't see why that test should fail if `LogConfig` is at the bottom of the structure. Running the test seems to work fine. How are you running the test and why does it fail?

Reply:
Wait, I'm confused. This is a TOML formatting failure? Or a test failure? Or something else?

Reply:
Known TOML formatting issue.

Reply:
Could you show me exactly how to get that error?

Reply:
Try to write_all to file when log field is after daemon field in struct definition

Reply:
Is it specific to `write_all` or can I use `toml::to_string(config)` or `toml::from_str`?

Reply:
Probably it isn't specific, but you `forest config dump` doesn't trigger it. Is it the only case we care about?

Reply:
Can we trigger it in another way with Forest? Or is it impossible to trigger it with Forest?

Reply:
How did you trigger it?

Reply:
I wrote a test where I dumped config to file.

I didn't try to trigger in with Forest.

Reply:
Please add that test to the PR. It sounds relevant.

Reply:
I've retested and everything went fine. Probably I missed something first time. So I returned `log` to its place.